### PR TITLE
added Router and Routes and Link packages, created Links,Components a…

### PR DESCRIPTION
**Pull request by ngiannotta84**
nd provisional routes

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
-        <p>Learn React</p>
-      </header>
+      <h2>SoundClone</h2>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="login" element={<Login />} />
+        <Route path="search" element={<Search />} />
+        <Route path="upload" element={<Upload />} />
+        <Route path="profile" element={<Profile />} />
```
Comment (PerryBaran) [inline src/components/App.js]: `path="profile" should be path="profile/:userId"`

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "../styles/navbar.css";
+import logo from "../assets/logosound.png";
+
+const Navbar = () => {
+  return (
+    <div className="navbar">
+      <img className="logo" src={logo} alt="logo soundcloud" />
```
Comment (PerryBaran) [inline src/components/Navbar.js]: wouldn't add the actual soundcloud logo, probably easier just to have this be a p tag with the text 'SOUNDCLONE'

File: src/index.js [hunk 3]
```diff
@@ -1,11 +1,14 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
+import { BrowserRouter } from "react-router-dom";
```
Comment (PerryBaran) [inline src/index.js]: import { BrowserRouter as Router }... then call Router instead of BrowserRouter

File: src/index.js [hunk 4]
```diff
@@ -1,11 +1,14 @@
...
 import App from "./components/App";
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+  <BrowserRouter>
```
Comment (PerryBaran) [inline src/index.js]: Best to move the Router to App.js

File: src/tests/App.test.js [hunk 5]
```diff
@@ -1,8 +1,9 @@
-import { render, screen } from '@testing-library/react';
-import App from '../components/App';
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import App from "../components/App";
 
-test('renders learn react link', () => {
+test("renders title", () => {
```
Comment (PerryBaran) [inline src/tests/App.test.js:5]: consider adding more tests, for example testing the navbar to see if it has all the text. You could also try looking into fireEvents and see if you can work out how test that clicking the links on the navbar works.

**Review by PerryBaran — CHANGES_REQUESTED** [review]

File: src/tests/App.test.js (hunk 5 above)
Comment (ngiannotta84) [reply src/tests/App.test.js:5]: Ok try to do that too

File: src/components/App.js (hunk 1 above)
Comment (ngiannotta84) [reply src/components/App.js]: ok no problem

File: src/components/Navbar.js (hunk 2 above)
Comment (ngiannotta84) [reply src/components/Navbar.js]: ok no problem

File: src/index.js (hunk 3 above)
Comment (ngiannotta84) [reply src/index.js]: Ok I am going to put it into App.js too as suggested

**Pull request closed without merging**